### ISSUE TITLE
Suggested means of checking error type of response doesn't work

### ISSUE THREAD

**Issue by chenwardT** [pseudonym117/Riot-Watcher#43]
You describe a means of checking status codes here https://github.com/pseudonym117/Riot-Watcher/issues/5#issuecomment-52090069 that doesn't work:

```
try:
    teams = riot_api.get_teams_for_summoner(summoner_id)
except LoLException as e:
    if e == error_404:
        ...
    else:
        ...
```

Sticking w/your example code, what I've found works is comparing the `error` attribute of the `e` object to the `error_` strings defined by riotwatcher, like so:

```
try:
    teams = riot_api.get_teams_for_summoner(summoner_id)
except LoLException as e:
    if e.error == error_404:
        ...
    else:
        ...
```

Can't speak to whether you intend for this kind of operation, but I thought it should be documented somewhere. If you are cool w/this method of determining error codes, I will submit a pull request w/an updated README as this is quite useful to know, especially in conjunction w/429 errors and the associated `Retry-After` header.

**Comment by pseudonym117** [comment]
ah, ok this was a side effect of pull request #35. I dont like this way of doing it but it may have to be the way to do it... old way worked with static objects being thrown, which isnt great, especially when the headers are included... 

anyways i fixed it so it should work the same way it did before. overloaded the equals operator to check for string equality. fixed as of commit 190751a

**Comment by pseudonym117** [comment]
although if you want to document how to get Retry-After, be my guest. I will merge that.
